### ISSUE TITLE
make mkvinfo GUI feature available as tab in MKVToolNix GUI

### ISSUE THREAD

**Issue by darealshinji** [mbunkus/mkvtoolnix#2104]
Request/idea: can you make the mkvinfo GUI feature available as a tab in MKVToolNix GUI? Just like the header editor.

**Comment by mbunkus** [comment]
Yes and no. No as in I cannot take the current GUI code and just wrap it in a tab as mkvinfo's code is making several assumptions that are plain incompatible with how MKVToolNix GUI works. That being said: I do have plans for re-writing that and including similar functionality in MKVToolNix GUI.

The long-term goal is to have a single GUI that includes for all the functionality the MKVToolNix package offers. That GUI will be MKVToolNix GUI, obviously, hence the name. However, info functionality has pretty much the lowest priority for me as there is a working GUI already (similar reason why I haven't included extraction functionality in MKVToolNix GUI yet; see #1026.

**Comment by mbunkus** [comment]
This project has been moved to [GitLab](https://gitlab.com/mbunkus/mkvtoolnix/). The corresponding issue is [here](https://gitlab.com/mbunkus/mkvtoolnix/issues/2104).
